image/remote.go

```diff
@@ -258,6 +261,8 @@ func (r *remote) Save() (string, error) {
 		return "", err
 	}
 
+	log.SetOutput(ioutil.Discard)
+	defer log.SetOutput(os.Stdout)
```

Comment:
The default value for the global logger is `os.Stderr` (and it doesn't appear to be possible to determine the current global `io.Writer` in the `log` package). Might be better to use `os.Stderr` here, but this also makes me feel like pushing this into the `main` package might be better.